### DISCUSSION TITLE
Is it possible to make a GET request in a CC and store it's output in a message?

### DISCUSSION THREAD

**Discussion by DvidPiDev** [yagpdb-cc/yagpdb-cc#311] [Q&A] [answered]
Just as the title says. I have a an API that returns a plaintext link to a random GIF on every refresh. I'm having trouble implementing it into YAGPDB.

**Comment by l-zeuch** [comment, marked as answer]
No, external calls to APIs are not possible with the custom command system, for security and rate-limit reasons.

**Comment by DvidPiDev** [comment]
Hi! I'd like to mention that I selfhost YAGPDB. Is that still possible?

**Reply by l-zeuch** [reply]
You can implement it yourself if you're comfortable with the codebase, but it's nothing that'll be added to mainline YAGPDB.
